### PR TITLE
Add photo section to people page

### PR DESCRIPTION
**Pull request by cdccollins**

### CODE REVIEW

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit ce314c1e70510809f9fc61d4b11b329369cfcdbd

https://deploy-preview-564--unboxed-co-production.netlify.com

File: source/assets/stylesheets/styles/_photo_gallery.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    line-height: 1.3em;
+  }
+}
+
+.photo-gallery-image {
+  overflow: auto;
+  min-height: 225px;
+}
+
+.gallery__image {
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_photo_gallery.scss]: I'm not sure if this class fits neatly into the BEM system. I'm getting confused with the class in the rule above.

Could we nest an `img` selector inside the above rule and lose the class here?

File: source/people.html.erb [hunk 2]
```diff
@@ -51,6 +67,27 @@ email_domain: "unboxed.co"
...
   </div>
+
+  <section class="photo-gallery" data-header-waypoint>
+    <p class="photo-gallery__introduction">
+      <%= current_page.data.inside_unboxed.title %>
+    </p>
+    <% current_page.data.inside_unboxed.content.each do |picture| %>
+      <div class="photo-gallery-container">
+        <div class="photo-gallery-image">
+          <%= image_tag(picture.image, alt: picture.image_description, class: "gallery__image", srcset: retina_srcset(picture.image)) %>
```
Comment (neilvanbeinum) [inline source/people.html.erb]: Could lose the `gallery__image` if we go with the above idea

File: source/people.html.erb [hunk 3]
```diff
@@ -51,6 +67,27 @@ email_domain: "unboxed.co"
...
+        </div>
+      </div>
+    <% end %>
+  </section>
+  <div class="job-alert">
+    <h2 class="job-alert__text">
+      Join Us
+      <br>
```
Comment (neilvanbeinum) [inline source/people.html.erb]: Do we need to use `<br>`s  here for spacing? I'd try two `h2` instead..

**Review by neilvanbeinum — APPROVED** [review]
Looks fab!

I've left a few ideas, but also approved.

FYI it's often a nice favour to the reviewer for PRs like this to include a screenshot of the new feature in the PR - just a tip!

File: source/assets/stylesheets/styles/_photo_gallery.scss (hunk 1 above)
Comment (cdccollins) [reply source/assets/stylesheets/styles/_photo_gallery.scss]: Yeah I spent a long time agonising over naming. I'll give your suggestions a go 😄

**Pull request merged**